src/staticOptions.ts

```diff
@@ -367,19 +373,14 @@ const staticOptions = asLanguageClientOptionsById({
   },
   swift: {
     // https://github.com/apple/sourcekit-lsp/blob/59b5e68f7f8408b5bc44bd47f71ef1afdc63e7a6/Editors/vscode/src/extension.ts#L22
-    documentSelector: [
-      'swift',
-      'cpp',
```

Comment:
why removing those 4 ? If they are not relevant anymore maybe we should remove the github link as well because it's confusing IMO

Reply:
We don't use it, there is no point of it and it can create issue in case of both swift and cpp LSP are started for some reasons :shrug:

Reply:
(link removed and amended)